content/en/blog/_posts/2021-06-18-Announcing-Kubernetes-Community-Group-Annual-Reports.md

```diff
@@ -0,0 +1,47 @@
+---
+layout: blog
+title: "Announcing Kubernetes Community Group Annual Reports"
+description: >
+  Introducing brand new Kubernetes Community Group Annual Reports for
+  Special Interest Groups and Working Groups.
+date: 2021-06-23T10:00:00-08:00
+slug: Announcing-Kubernetes-Community-Group-Annual-Reports
+---
+
+**Authors:** Divya Mohan
+
+Given the growth and scale of the Kubernetes project, the existing reporting mechanisms were proving to be inadequate and challenging.
+Kubernetes is a large open source project. With over 100000 commits just to the main k/kubernetes repository, hundreds of other code
+repositories in the project, and thousands of contributors, there's a lot going on. In fact, there are 37 contributor groups at the time of
+writing. We also value all forms of contribution and not just code changes.
+
+With that context in mind, the challenge of reporting on all this activity was a call to action for exploring better options. Therefore
+inspired by the Apache Software Foundation’s [open guide to PMC Reporting](https://www.apache.org/foundation/board/reporting) and the
+[CNCF project Annual Reporting](https://www.cncf.io/cncf-annual-report-2020/), the Kubernetes project is proud to announce the 
+**Kubernetes Community Group Annual Reports for Special Interest Groups (SIGs) and Working Groups (WGs)**. In its flagship edition, 
+the [2020 Summary report](https://github.com/kubernetes/steering/blob/master/reports/summary-2020.md) focuses on bettering the 
+Kubernetes ecosystem by assessing and promoting the healthiness of the groups within the upstream community. 
```

Comment:
```suggestion
the [2020 Summary report](https://www.cncf.io/kubernetes-community-annual-report-2020/) focuses on bettering 
the Kubernetes project by assessing and promoting the healthiness of the groups within the community. 
```

switched to CNCF report
cc @parispittman do you think we should switch it or keep it pointed at the GH report?

Reply:
@parispittman : Will await your response on this. I have committed the other suggestions made by y'all :)

Reply:
I prefer linking to https://www.cncf.io/kubernetes-community-annual-report-2020/

Reply:
+1 to @sftim